continuous_main.py

```diff
@@ -36,7 +37,11 @@
 plt.legend()
 plt.show()
 
-y0_ = model.tau(x_[:,0]).numpy()
+device = next(model.parameters()).device
+
+model.eval()
+with torch.no_grad():
+    y0_ = model.tau(torch.tensor(x_[:, 0], dtype=torch.float, device=device)).cpu().numpy()
```

Comment:
This added section of code that effectively serves to project the data seems a bit wordy to me. Could you simplify it, or wrap it up in some function so that the API remains simple to use? We want the API to be as high-level as possible (at the level a neuroscientist would like to use), without going into the deep learning implementation details. This would make it less confusing to the new user.

Try to mirror the older API (using TensorFlow) as closely as possible, or wrap up the other lines within already existing or, if necessary, a new function.